# Análisis 2da Encuesta de inserción de científicos en Chile

**Autor:** Roberto Muñoz <br />
**E-mail:** <rmunoz@metricarts.com> <br />
**Github:** <https://github.com/rpmunoz> <br />

Análisis preliminar de la [2da Encuesta de inserción de científicos](https://www.anip.cl/encuesta-insercion-2018) llevada a cabo por el Centro de estudios de la ANIP. 

In [1]:
import os
import shutil
import numpy as np
import pandas as pd
from scipy import stats, integrate

import matplotlib.pyplot as plt
import matplotlib.ticker as ticker
import seaborn as sns
from IPython.core.display import display, HTML

%matplotlib inline
params = {'legend.fontsize': 'large',
          'figure.figsize': (8, 6),
         'axes.labelsize': 'large',
         'axes.titlesize':'large',
         'xtick.labelsize':'large',
         'ytick.labelsize':'large',
         'axes.titlepad': 16,
         'axes.titlesize': 20,
         'axes.labelpad': 16,
         'axes.labelsize': 16,
         'lines.linewidth': 3,
         'lines.markersize': 8,
         'xtick.labelsize': 12,
         'ytick.labelsize': 12}
plt.rcParams.update(params)

pd.set_option('display.max_columns', None)

## 1. Cargamos el archivo de datos original

Cargamos el archivos de datos que contiene todas las encuestas válidas. 

In [2]:
encuesta_full_file="../encuesta/E2018-full-answers.csv"

encuesta_full_df=pd.read_csv(encuesta_full_file)
encuesta_full_df.head()

ID de la respuesta  Fecha en que inició Fecha de la ultima accion  \
0                  41  2018-01-19 12:21:56       2018-01-19 13:02:59   
1                  42  2018-01-19 12:24:06       2018-01-19 12:29:59   
2                  43  2018-01-19 12:26:32       2018-01-19 12:34:04   
3                  44  2018-01-19 12:28:30       2018-01-19 12:36:47   
4                  45  2018-01-19 12:37:03       2018-01-19 12:44:43   

  ¿Cuál es el grado académico más alto que has obtenido?  \
0                                    Doctorado (PhD)       
1                                    Doctorado (PhD)       
2                                     Magíster (MSc)       
3                                    Doctorado (PhD)       
4                                    Doctorado (PhD)       

  ¿En qué disciplina y área clasificarías tu pregrado?  \
0             Ciencias Naturales:Ciencias Biológicas     
1                  Humanidades:Lenguaje y Literatura     
2         Ciencias Sociales:Ciencias de la Educación     
3               Ciencias Naturales:Ciencias Químicas     
4                     Ciencias Naturales:Matemáticas     

  ¿En qué disciplina y área clasificarías tu pregrado? [Otro]  \
0                                                NaN            
1                                                NaN            
2                                                NaN            
3                                                NaN            
4                                                NaN            

  ¿En qué año obtuviste tu pregrado?  \
0                               2003   
1                               2004   
2                               2014   
3                               2012   
4                               2010   

  ¿En qué disciplina y área clasificarías tu {postgrado}?  \
0             Ciencias Naturales:Ciencias Biológicas        
1                  Humanidades:Lenguaje y Literatura        
2  Ciencias Médicas y de la Salud:Ciencias de la ...        
3               Ciencias Naturales:Ciencias Químicas        
4                     Ciencias Naturales:Matemáticas        

  ¿En qué disciplina y área clasificarías tu {postgrado}? [Otro]  \
0                                                NaN               
1                                                NaN               
2                                                NaN               
3                                                NaN               
4                                                NaN               

  ¿En qué año obtuviste tu {postgrado}?  \
0                                  2017   
1                                  2017   
2                                  2018   
3                                  2016   
4                                  2017   

  ¿En qué lugar realizaste tu {postgrado}?  \
0                                    Chile   
1                                   España   
2                                    Chile   
3                                    Chile   
4                                  Francia   

   En total, ¿cuántos artículos revisados por pares (peer-reviewed) has publicado en los últimos 5 años?  \
0                                                 10                                                       
1                                                  3                                                       
2                                                  3                                                       
3                                                  4                                                       
4                                                  1                                                       

  ¿Cómo financiaste tus estudios de {postgrado}? [Fondos personales (e.g., ahorros, aporte familiar, etc.)]  \
0                                                 No                                                          
1                                            

In [3]:
n_registros=len(encuesta_full_df)
print("Numbero de registros", n_registros)

Numbero de registros 752


In [4]:
n_campos=len(list(encuesta_full_df.keys()))
print("Numbero de campos", n_campos)

Numbero de campos 132


In [5]:
encuesta_full_df.dtypes.to_frame().transpose()

ID de la respuesta Fecha en que inició Fecha de la ultima accion  \
0              int64              object                    object   

  ¿Cuál es el grado académico más alto que has obtenido?  \
0                                             object       

  ¿En qué disciplina y área clasificarías tu pregrado?  \
0                                             object     

  ¿En qué disciplina y área clasificarías tu pregrado? [Otro]  \
0                                             object            

  ¿En qué año obtuviste tu pregrado?  \
0                             object   

  ¿En qué disciplina y área clasificarías tu {postgrado}?  \
0                                             object        

  ¿En qué disciplina y área clasificarías tu {postgrado}? [Otro]  \
0                                             object               

  ¿En qué año obtuviste tu {postgrado}?  \
0                                object   

  ¿En qué lugar realizaste tu {postgrado}?  \
0                                   object   

  En total, ¿cuántos artículos revisados por pares (peer-reviewed) has publicado en los últimos 5 años?  \
0                                              int64                                                      

  ¿Cómo financiaste tus estudios de {postgrado}? [Fondos personales (e.g., ahorros, aporte familiar, etc.)]  \
0                                             object                                                          

  ¿Cómo financiaste tus estudios de {postgrado}? [Trabajo durante el estudio (e.g., pasantías, ayudantías, etc.)]  \
0                                             object                                                                

  ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales nacionales (e.g., becas Conicyt, Becas Chile, premios, etc.)]  \
0                                             object                                                                                   

  ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales extranjeros (e.g., becas o premios de universidad de estudios)]  \
0                                             object                                                                                     

  ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en mi área de estudio]  \
0                                             object                                                    

  ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en otra área de estudio]  \
0                                             object                                                      

  ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Pasantía(s) en otros lugares de estudio o centros de investigación]  \
0                                             object                                                                                    

  ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo en mi lugar de estudio, sin relación con mis estudios]  \
0                                             object                                                                               

  ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo fuera de mi lugar de estudio]  \
0                                             object                                                      

  ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo remoto de cualquier tipo]  \
0                                             object                                                  

  ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Otro]  \
0                                             object                      

  ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Beca o premio (por favor indica el nombre de la beca o premio)]  \
0                                             object                        

## 2. Análisis exploratorio y limipieza de datos

Haremos un análisis exploratorio de los datos. Nos concetraremos en los últimos campos de la encuesta para revisar errores de arrastre o tipeo.

In [6]:
field_names_full=['¿Cuál es tu género?',
             '¿Cuál es tu edad?',
             '¿Cuál es tu nacionalidad? En caso de tener más de una nacionalidad, por favor escoge tu país de origen (donde naciste).']

Primero buscamos cuales de estos campos contienen valores NaN

In [7]:
encuesta_full_df[field_names_full].isnull().sum()

¿Cuál es tu género?                                                                                                         1
¿Cuál es tu edad?                                                                                                          34
¿Cuál es tu nacionalidad? En caso de tener más de una nacionalidad, por favor escoge tu país de origen (donde naciste).     0
dtype: int64

Luego buscamos los valores únicos de estos campos para encontrar posibles errores

In [8]:
for field_name in field_names_full:
    print(encuesta_full_df[field_name].unique())

['Masculino' 'Femenino' 'Prefiero no responder'
 ' o buscan técnicos que salen más económicos para que aprendan lo básico (y no necesariamente ir subiendo sus puestos) o buscan doctorados expertos en un área. Estando uno en un nivel intermedio (o avanzado en un área ahora sin oferta) es difícil encontrar algo.'
 'Otro (no binario)'
 'Creo que es muy deficiente. Faltan mecanismos para integrarse, recursos para financiar las iniciativas de inserción estatales y fomentar la participción del mundo privado. Quizás se pueda fomentar el uso de un fondo tipo Ley Valdés de donaciones, para que las empresas financien distintos tipos de investigación. '
 nan ' con contrato y no con boleta.']
['40' '37' '36' '31' '32' '43' '42' '41' '34' '39' '33' '27' '29' '30' '49'
 '38' '48' '35' nan '44' '52' '25' '45' '28' '57' 'Femenino' '50' '46' '26'
 '61' '58' '47' 'Masculino' '68' '53']
['Chile' 'Francia' 'España' 'Colombia' 'Bélgica' 'Uruguay' 'Venezuela'
 'Ecuador' 'Brasil' 'Reino Unido' 'Alemania' 'Ca

Podemos notar que existe un existen respuestas incorrectas en los campos Genero, Edad y Nacionalidad. Explores cuales ID son las respuestas afectadas

Haremos una copia del archivo csv y buscaremos esos registros e

In [9]:
encuesta_full_file="../encuesta/E2018-full-answers.csv"
encuesta_clean_file="../encuesta/E2018-full-answers-clean.csv"

if not os.path.exists(encuesta_clean_file):
    shutil.copyfile(encuesta_full_file, encuesta_clean_file)

In [10]:
encuesta_full_df[encuesta_full_df['¿Cuál es tu género?']==' o buscan técnicos que salen más económicos para que aprendan lo básico (y no necesariamente ir subiendo sus puestos) o buscan doctorados expertos en un área. Estando uno en un nivel intermedio (o avanzado en un área ahora sin oferta) es difícil encontrar algo.']

ID de la respuesta  Fecha en que inició Fecha de la ultima accion  \
197                 334  2018-01-24 23:37:08       2018-01-24 23:45:55   

    ¿Cuál es el grado académico más alto que has obtenido?  \
197                                     Magíster (MSc)       

    ¿En qué disciplina y área clasificarías tu pregrado?  \
197    Ingeniería y Tecnología:Biotecnología Ambiental     

    ¿En qué disciplina y área clasificarías tu pregrado? [Otro]  \
197                                                NaN            

    ¿En qué año obtuviste tu pregrado?  \
197                               2017   

    ¿En qué disciplina y área clasificarías tu {postgrado}?  \
197             Ingeniería y Tecnología:Nanotecnología        

    ¿En qué disciplina y área clasificarías tu {postgrado}? [Otro]  \
197                                                NaN               

    ¿En qué año obtuviste tu {postgrado}?  \
197                                  2017   

    ¿En qué lugar realizaste tu {postgrado}?  \
197                                    Chile   

     En total, ¿cuántos artículos revisados por pares (peer-reviewed) has publicado en los últimos 5 años?  \
197                                                  0                                                       

    ¿Cómo financiaste tus estudios de {postgrado}? [Fondos personales (e.g., ahorros, aporte familiar, etc.)]  \
197                                                 Sí                                                          

    ¿Cómo financiaste tus estudios de {postgrado}? [Trabajo durante el estudio (e.g., pasantías, ayudantías, etc.)]  \
197                                                 No                                                                

    ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales nacionales (e.g., becas Conicyt, Becas Chile, premios, etc.)]  \
197                                                 No                                                                                   

    ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales extranjeros (e.g., becas o premios de universidad de estudios)]  \
197                                                 No                                                                                     

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en mi área de estudio]  \
197                                                NaN                                                    

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en otra área de estudio]  \
197                                                NaN                                                      

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Pasantía(s) en otros lugares de estudio o centros de investigación]  \
197                                                NaN                                                                                    

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo en mi lugar de estudio, sin relación con mis estudios]  \
197                                                NaN                                                                               

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo fuera de mi lugar de estudio]  \
197                                                NaN                                                      

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo remoto de cualquier tipo]  \
197                                                NaN                                                  

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Otro]  \
197                                                NaN                      

    ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Beca o premio (por favor indica el nombre de la beca o

- Corregimos la respuesta abierta del ID 334.

Aquí el error ocurre en el campo 'Finalmente, ¿cuál es tu opinión sobre el sistema de ciencia y tecnología en Chile, y sobre el proceso de inserción de científicos en Chile?', donde la persona responde usando coma y el sistema separa la respuesta en dos campos. Agregamos doble comillas

In [11]:
encuesta_full_df[encuesta_full_df['¿Cuál es tu género?']=='Creo que es muy deficiente. Faltan mecanismos para integrarse, recursos para financiar las iniciativas de inserción estatales y fomentar la participción del mundo privado. Quizás se pueda fomentar el uso de un fondo tipo Ley Valdés de donaciones, para que las empresas financien distintos tipos de investigación. ']

ID de la respuesta  Fecha en que inició Fecha de la ultima accion  \
303                 477  2018-01-25 22:27:41       2018-01-25 22:38:21   

    ¿Cuál es el grado académico más alto que has obtenido?  \
303                                    Doctorado (PhD)       

    ¿En qué disciplina y área clasificarías tu pregrado?  \
303                 Humanidades:Historia y Arqueología     

    ¿En qué disciplina y área clasificarías tu pregrado? [Otro]  \
303                                                NaN            

    ¿En qué año obtuviste tu pregrado?  \
303                               2005   

    ¿En qué disciplina y área clasificarías tu {postgrado}?  \
303                 Humanidades:Historia y Arqueología        

    ¿En qué disciplina y área clasificarías tu {postgrado}? [Otro]  \
303                                                NaN               

    ¿En qué año obtuviste tu {postgrado}?  \
303                                  2016   

    ¿En qué lugar realizaste tu {postgrado}?  \
303                Estados Unidos de América   

     En total, ¿cuántos artículos revisados por pares (peer-reviewed) has publicado en los últimos 5 años?  \
303                                                  2                                                       

    ¿Cómo financiaste tus estudios de {postgrado}? [Fondos personales (e.g., ahorros, aporte familiar, etc.)]  \
303                                                 No                                                          

    ¿Cómo financiaste tus estudios de {postgrado}? [Trabajo durante el estudio (e.g., pasantías, ayudantías, etc.)]  \
303                                                 No                                                                

    ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales nacionales (e.g., becas Conicyt, Becas Chile, premios, etc.)]  \
303                                                 Sí                                                                                   

    ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales extranjeros (e.g., becas o premios de universidad de estudios)]  \
303                                                 Sí                                                                                     

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en mi área de estudio]  \
303                                                NaN                                                    

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en otra área de estudio]  \
303                                                NaN                                                      

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Pasantía(s) en otros lugares de estudio o centros de investigación]  \
303                                                NaN                                                                                    

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo en mi lugar de estudio, sin relación con mis estudios]  \
303                                                NaN                                                                               

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo fuera de mi lugar de estudio]  \
303                                                NaN                                                      

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo remoto de cualquier tipo]  \
303                                                NaN                                                  

    ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Otro]  \
303                                                NaN                      

    ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Beca o premio (por favor indica el nombre de la beca o

- La respuesta del registro 477 parece bien, imprimiré en pantalla la respuesta detallada

Este error es más sútil, pues en el campo '¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario]' la persona respondió dos frases y las separó por coma, lamentablemente el sistema las separó en dos columnas. Arreglamos el csv agregando doblo comilla.

In [166]:
encuesta_full_df[encuesta_full_df['¿Cuál es tu género?']==' con contrato y no con boleta.']

Empty DataFrame
Columns: [ID de la respuesta, Fecha en que inició, Fecha de la ultima accion, ¿Cuál es el grado académico más alto que has obtenido?, ¿En qué disciplina y área clasificarías tu pregrado?, ¿En qué disciplina y área clasificarías tu pregrado? [Otro], ¿En qué año obtuviste tu pregrado?, ¿En qué disciplina y área clasificarías tu {postgrado}?, ¿En qué disciplina y área clasificarías tu {postgrado}? [Otro], ¿En qué año obtuviste tu {postgrado}?, ¿En qué lugar realizaste tu {postgrado}?, En total, ¿cuántos artículos revisados por pares (peer-reviewed) has publicado en los últimos 5 años?, ¿Cómo financiaste tus estudios de {postgrado}? [Fondos personales (e.g., ahorros, aporte familiar, etc.)], ¿Cómo financiaste tus estudios de {postgrado}? [Trabajo durante el estudio (e.g., pasantías, ayudantías, etc.)], ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales nacionales (e.g., becas Conicyt, Becas Chile, premios, etc.)], ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales extranjeros (e.g., becas o premios de universidad de estudios)], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en mi área de estudio], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en otra área de estudio], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Pasantía(s) en otros lugares de estudio o centros de investigación], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo en mi lugar de estudio, sin relación con mis estudios], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo fuera de mi lugar de estudio], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo remoto de cualquier tipo], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Otro], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Beca o premio (por favor indica el nombre de la beca o premio)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Crédito de un banco (por favor indica el nombre del banco)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario].1, ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Financiamiento de empleador (por favor indica el nombre del empleador)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario].2, ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Otro tipo de fondos (por favor indica qué tipo)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario].3, ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Beca o premio (por favor indica el nombre de la beca o premio)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Crédito de un banco (por favor indica el nombre del banco)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario].1, ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Financiamiento de empleador (por favor indica el nombre del empleador)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario].2, ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Otro tipo de fondos (por favor indica qué tipo)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario].3, ¿Estás realizando o has realizado algún postdoctorado?, ¿En qué disciplina y área clasificarías tu postdoctorado?, ¿En qué disciplina y área clasificarías tu postdoctorado? [Otro], ¿En qué año terminaste tu postdoctorado? Si actuamente estás haciendo un postdoctorado, por favor responde "Estoy haciendo un postdoctorado ahora"., ¿En qué lugar realizaste (o estás realizando) tu postdoctorado?, ¿Has terminado más de un postdoctorado?, ¿Cuántos postdoctorados has realizado?, ¿Estás en Chile ahora?, ¿Tuviste que renunciar a

- Corregimos la respuesta abierta del ID 849.

Aquí el error ocurre en el campo 'Finalmente, ¿cuál es tu opinión sobre el sistema de ciencia y tecnología en Chile, y sobre el proceso de inserción de científicos en Chile?', donde la persona responde usando coma y el sistema separa la respuesta en dos campos. Agregamos doble comillas

In [167]:
encuesta_full_df[encuesta_full_df['¿Cuál es tu género?'].isnull()]

Empty DataFrame
Columns: [ID de la respuesta, Fecha en que inició, Fecha de la ultima accion, ¿Cuál es el grado académico más alto que has obtenido?, ¿En qué disciplina y área clasificarías tu pregrado?, ¿En qué disciplina y área clasificarías tu pregrado? [Otro], ¿En qué año obtuviste tu pregrado?, ¿En qué disciplina y área clasificarías tu {postgrado}?, ¿En qué disciplina y área clasificarías tu {postgrado}? [Otro], ¿En qué año obtuviste tu {postgrado}?, ¿En qué lugar realizaste tu {postgrado}?, En total, ¿cuántos artículos revisados por pares (peer-reviewed) has publicado en los últimos 5 años?, ¿Cómo financiaste tus estudios de {postgrado}? [Fondos personales (e.g., ahorros, aporte familiar, etc.)], ¿Cómo financiaste tus estudios de {postgrado}? [Trabajo durante el estudio (e.g., pasantías, ayudantías, etc.)], ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales nacionales (e.g., becas Conicyt, Becas Chile, premios, etc.)], ¿Cómo financiaste tus estudios de {postgrado}? [Fondos no personales extranjeros (e.g., becas o premios de universidad de estudios)], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en mi área de estudio], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Ayudantía(s) en otra área de estudio], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Pasantía(s) en otros lugares de estudio o centros de investigación], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo en mi lugar de estudio, sin relación con mis estudios], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo fuera de mi lugar de estudio], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Trabajo remoto de cualquier tipo], ¿Qué tipo de trabajo desempeñaste para financiar tus estudios? [Otro], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Beca o premio (por favor indica el nombre de la beca o premio)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Crédito de un banco (por favor indica el nombre del banco)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario].1, ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Financiamiento de empleador (por favor indica el nombre del empleador)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario].2, ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Otro tipo de fondos (por favor indica qué tipo)], ¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario].3, ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Beca o premio (por favor indica el nombre de la beca o premio)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Crédito de un banco (por favor indica el nombre del banco)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario].1, ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Financiamiento de empleador (por favor indica el nombre del empleador)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario].2, ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Otro tipo de fondos (por favor indica qué tipo)], ¿Con qué tipo de fondos extranjeros financiaste tu {postgrado}? [Comentario].3, ¿Estás realizando o has realizado algún postdoctorado?, ¿En qué disciplina y área clasificarías tu postdoctorado?, ¿En qué disciplina y área clasificarías tu postdoctorado? [Otro], ¿En qué año terminaste tu postdoctorado? Si actuamente estás haciendo un postdoctorado, por favor responde "Estoy haciendo un postdoctorado ahora"., ¿En qué lugar realizaste (o estás realizando) tu postdoctorado?, ¿Has terminado más de un postdoctorado?, ¿Cuántos postdoctorados has realizado?, ¿Estás en Chile ahora?, ¿Tuviste que renunciar a

- Corregimos la respuesta abierta del ID 817.

Este error es más sútil, pues en el campo '¿Con qué tipo de fondos nacionales financiaste tu {postgrado}? [Comentario]' la persona respondió dos frases y las separó por coma, lamentablemente el sistema las separó en dos columnas. Arreglamos el csv agregando doble comilla.